cmd/keymasterd/templateData.go

```diff
@@ -69,7 +69,7 @@ const loginFormText = `
 	{{template "login_pre_password" .}}
         <form enctype="application/x-www-form-urlencoded" action="/api/v0/login" method="post">
             <p>Username: <INPUT TYPE="text" NAME="username" SIZE=18></p>
-            <p>Password: <INPUT TYPE="password" NAME="password" SIZE=18></p>
+            <p>Password: <INPUT TYPE="password" NAME="password" SIZE=18  autocomplete="off"></p>
```

Comment:
Will this prevent saved passwords from being auto-filled?

Reply:
By default yes.. you can always ask you password manager to do it for you.

Reply:
Let's talk about this offline. This doesn't sound like a user-positive change.

Reply:
Setting autocomplete="off" on fields has two effects:

    It tells the browser not to save data inputted by the user for later autocompletion on similar forms, though heuristics for complying vary by browser.
    It stops the browser from caching form data in the session history. When form data is cached in session history, the information filled in by the user is shown in the case where the user has submitted the form and clicked the Back button to go back to the original form page.